### ISSUE TITLE
[DOC] ML Troubleshooting page

### ISSUE THREAD

**Issue by dtaivpp** [opensearch-project/documentation-website#4867]
**What do you want to do?**
 At the moment it can be challenging to understand what to do next when something goes wrong with your ML models. We should have a dedicated section that describes some common issues and what to do when you encounter them. 
 
- [ ] Request a change to existing documentation
- [X] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 We need some concrete steps for users to understand what is going on with their ML Models and how to debug them. 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
For example, I had a ML Model that failed to upload and I was unable to find how to fix it after seeing the failed task. I should have deleted and re-created the model.

**Comment by hdhalter** [comment]
@dtaivpp - Do you have any suggestions for who can help write this content?

**Comment by dtaivpp** [comment]
Yeah, @dylan-tong-aws and I will probably take the bulk of it. I am going to put these on a meta issue over in the ML-Commons repo to see about getting some traction.

**Comment by hdhalter** [comment]
Reassigning to @kolchfa-aws , since it is machine learning-related.

**Comment by kolchfa-aws** [comment]
Unassigning myself so this issue can be picked up by someone from the community who went through this process using the new documentation. Thanks!

**Comment by natebower** [comment]
Closing as stale
